web-components/chart/chart.tsx

```diff
@@ -260,6 +260,11 @@ export class CobChart {
     selectElem.className = 'sel-f';
     selectArrow.appendChild(selectElem);
 
+    // Set the default selection if set
+    this.config.boston.defaultSelection
```

Comment:
One other cute way of doing this is:

`selectElem.value = this.config.boston.defaultSelection || this.selectOptions[0];`

due to how JavaScript short-circuits boolean operators.

Reply:
love it! thanks for the tip